js/apps/shared/filter-helper.service.js

```diff
@@ -25,25 +25,25 @@
 			return !!matches.length;
 		};
 
-		//TODO: FILTERS MUST BE A STRING???
+		// TODO: FILTERS MUST BE A STRING???
 		const getFiltersFromString = (filterStr, isDate) => {
-			if(!filterStr) return [];
+			if (!filterStr) return [];
 
-			return filterStr && filterStr.indexOf(',') > -1 ?  
+			return filterStr.indexOf(',') > -1 ?
 				(
-					isDate ? 
-						filterStr : 
+					isDate ?
+						filterStr :
 						filterStr.split(',')
-				) : 
-				[ filterStr ];
+				) :
+				[filterStr];
 		};
 
 		const getQueryParams = () => $location.search();
 
 		const getQueryParamValuesByKey = (queryParams, key, isDate) => {
 			return Object.hasOwnProperty.call(queryParams, key) ?
 				getFiltersFromString(queryParams[key], isDate) :
-				(isDate ? "" : []);
+				(isDate ? '' : []);
```

Comment:
Are we standardizing on using single quotes for strings?  Is there a lint check for this?

Reply:
@ascire you are right, i'm not sure how this got in here to be honest. This file will be in another pr, so ill fix it there.